src/components/LeftPanel/index.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+import { SideNav, SideNavItems, SideNavLink } from '@carbon/react';
+
+import * as Icons from '@carbon/icons-react';
```

Comment:
I'm not sure if this is a good idea. I would say that is clearer if we import only the icons that we're going to use.